module/zfs/dbuf.c

```diff
@@ -1152,7 +1152,7 @@ dbuf_whichblock(const dnode_t *dn, const int64_t level, const uint64_t offset)
 		 * 2^(level*(indblkshift - SPA_BLKPTRSHIFT)).
 		 *
 		 * Thus, the level n blkid is: offset /
-		 * ((2^datablkshift)*(2^(level*(indblkshift - SPA_BLKPTRSHIFT)))
+		 * ((2^datablkshift)*(2^(level*(indblkshift - SPA_BLKPTRSHIFT))))
```

Comment:
nit: `./module/zfs/dbuf.c: 1155: line > 80 characters`

I'd suggest removing the spaces are the minus sign to fix the line length, or wrapping the lines differently.